matcher/service/openreview_interface.py

```diff
@@ -130,6 +132,10 @@ def match_group(self):
     def reviewers(self):
         return self.match_group.members
 
+    @property
+    def map_reviewers_to_indexes(self):
+        return self.match_group.members    
```

Comment:
this is not a map is an array of values, right?

Reply:
Right, it is fixed now.